core/trino-spi/src/main/java/io/trino/spi/eventlistener/ColumnMaskReferenceInfo.java

```diff
@@ -13,27 +13,19 @@
  */
 package io.trino.spi.eventlistener;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
+import static java.util.Objects.requireNonNull;
 
 /**
  * This class is JSON serializable for convenience and serialization compatibility is not guaranteed across versions.
  */
-public class ColumnMaskReferenceInfo
-        extends FilterMaskReferenceInfo
+public record ColumnMaskReferenceInfo(@Override String expression, @Override String targetCatalogName, @Override String targetSchemaName, @Override String targetTableName, String targetColumnName)
+        implements FilterMaskReferenceInfo
 {
-    private final String columnName;
-
-    @JsonCreator
-    public ColumnMaskReferenceInfo(String maskExpression, String targetCatalogName, String targetSchemaName, String targetTableName, String targetColumnName)
-    {
-        super(maskExpression, targetCatalogName, targetSchemaName, targetTableName);
-        this.columnName = targetColumnName;
-    }
-
-    @JsonProperty
-    public String getTargetColumnName()
-    {
-        return columnName;
+    public ColumnMaskReferenceInfo {
```

Comment:
Nit: brace on next line

Same for the others

Reply:
Done